packages/shell-api/src/field-level-encryption.ts

```diff
@@ -408,27 +407,27 @@ export class KeyVault extends ShellApiWithMongoClass {
   @apiVersions([1])
   async deleteKey(keyId: BinaryType): Promise<DeleteResult | Document> {
     assertArgsDefinedType([keyId], [true], 'KeyVault.deleteKey');
-    return this._clientEncryption._libmongocrypt.deleteKey(keyId);
+    return await this._clientEncryption._libmongocrypt.deleteKey(keyId);
```

Comment:
Similar to a comment above - this was the easiest way to make linting, vscode and the compiler all happy at once about the fact that this was an async function with no await.